lib/ood_core/job/adapters/sge/batch.rb

```diff
@@ -62,16 +62,19 @@ def load_drmaa(libdrmaa_path)
   # @param owner [#to_s] the owner or owner list
   # @return [Array<OodCore::Job::Info>]
   def get_all(owner: nil)
-    listener = QstatXmlRListener.new
-    argv = ['qstat', '-r', '-xml']
-    argv.concat ['-u', owner] unless owner.nil?
-    REXML::Parsers::StreamParser.new(call(*argv), listener).parse
-
-    listener.parsed_jobs.map{
-      |job_hash| OodCore::Job::Info.new(
-        **post_process_qstat_job_hash(job_hash)
-      )
-    }
```

Comment:
This `map` call I did refactor a little bit just to be more in line with what we typically do elsewhere (better newlines and `do end` instead of `{}` ).

```ruby
      listener.parsed_jobs.map do |job_hash| 
        OodCore::Job::Info.new(**post_process_qstat_job_hash(job_hash))
      end
```